docs/QUICK_START.md

```diff
@@ -301,6 +309,14 @@ System:
   Boot into FW: supported
 ```
 
+> ⚠️  If you used `--microsoft` while enrolling the keys, you might want
+> to check that the Secure Boot Forbidden Signature Database (dbx) is not
+> empty.
+> A quick and dirty way is by checking the file size of
+> `/sys/firmware/efi/efivars/dbx-*`.
```

Comment:
We should refill it via the published dbx by UEFI/MSFT

Reply:
That's maybe a thing for `sbctl`

Reply:
I agree with the fact that maybe `sbctl` should include an additional command to refill the dbx.
This could use <https://uefi.org/revocationlistfile>.